walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -0,0 +1,345 @@
+
+[time=6]
+== Conflict resolution functionalities
+
+Data Conflicts can happen when resource we modified was also modified on server.
+DataSync Framework enables multiple strategies to deal with conflicts. 
+By default Showcase will resolve conflicts by applying all server side changes on top of the client.
+In case of direct conflicts client field value will be used. 
+To simulate conflict we need to go offline and make edits in two instance of the application at the same time.
+
+=== Steps
+
+. Use one of the existing task or create new one that will be visible on two devices.
+. Go offline 
+. Edit Task on first instance
+. Go to second instance of the application
+. Edit Task on second instance
+. At this point we should have server side updated by second instance of the application
+and first application not knowing about that change because it went offline.
+. This operation will result in conflict. 
+. Once we make first client online again - data conflict will occur.
+We should see popup and actuall server side data being replicated back to the client (conflict was resolved automatically)
+
+. Use the “Manage Tasks” page in the showcase client app.
+. Try create one task in one device, and it should show up immediately in another.
+. Try bring one client offline (, perform some actions, bring it online and you should see the changes synced to another app straight away.
+. To create conflicts, try bringing both clients offline, and modify the same item. The last instance of app that is online will receive conflicts.
+
+[time=35]
+== [Optional] Check the Data Synchonization Audit Logs
+
+You can use the OpenShift logging feature to see the audit logs that are generated by the syncserver app.
+
+=== Steps
+
+. Find out the URL of the Kibana dashboard of your OpenShift cluster:
+.. Run the following commands on the bastion server (as root user):
++
+----
+oc project openshift-logging
+oc get route
+----
++
+.. You should see there is a route called `logging-kibana`. Copy the `Host` value of that route and open it in your browser. You should see the Kibana dashboard page.
+. Filter out the audit log messages in Kibana. 
+.. On the `Discover` page, there should be a dropdown you can select on this page. Choose the namespace where the syncserver app is deployed to, and take a look at the messages.
+.. You should see a lot of messages, try expand one and see what the message looks like.
+.. In the `Search` field on the top, enter `tag:AUDIT` and you should be able to see the audit logs generated by the app. If you can't see any message, go to the GraphqQL playground page and execute some queries and then try search again.
+.. [Optional] Follow the steps descibed in link:https://github.com/aerogear/apollo-voyager-server/blob/master/doc/guides/metrics-and-audit-logs.md#importing-kibana-saved-objects:[Importing Kibana Saved Objects guide] and try importing a dashboard template for the sync app, and view the imported dashboard.
```

Comment:
Not sure if we should migrate those steps into this doc. I have tried to follow this and it takes some time to get that applied